notebooks/111-yolov5-quantization-migration/111-yolov5-quantization-migration.ipynb

```diff
@@ -72,13 +72,13 @@
      "output_type": "stream",
```

Comment:
<p>OpenVINO Intermediate Representation (IR) model format with FP32 precision generated by Model Optimizer. -&gt; OpenVINO Intermediate Representation (IR) model format with FP32 precision generated by <a href="https://docs.openvino.ai/2023.0/openvino_docs_model_processing_introduction.html" target="_blank">OpenVINO model conversion API</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Fixed, thank you!</p>